packages/ts-config-single-spa/tsconfig.json

```diff
@@ -14,6 +14,7 @@
       "es2016.array.include",
       "es2018"
     ],
-    "noEmit": true
+    "declaration": true,
```

Comment:
These changes are what makes this a breaking change - previously no types emitted, now types are emitted